src/pages/projects.jsx

```diff
@@ -15,48 +15,86 @@ const StyledH1 = styled.h1`
     margin-bottom: 20px;
 `;
 
+const StyledP1 = styled.p`
+    font-size: 25px;
+    margin-bottom: 100px;
+`;
+
 const StyledP = styled.p`
     font-size: 25px;
+    margin-bottom: 20px;
 `;
 
 const StyledImg = styled.img`
     border-style: solid;
     border-color: black;
-    margin-bottom: 30px;
+    margin-bottom: 80px;
+`;
+
+const StyledButt = styled.button`

```

Comment:
I know this is just a joke, but it does make the code harder to read. Without seeing how this is used, I can't tell what it's used for. Can you rename this to `ButtonStyling`?